app/qml/components/MMSelectableToolbar.qml

```diff
@@ -0,0 +1,41 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import Qt5Compat.GraphicalEffects
+
+Rectangle {
+  id: control
+
+  signal clicked
+
+  required property alias model
+  property alias index: buttonView.currentIndex
+
+  anchors {
+    left: parent.left
+    right: parent.right
+    bottom: parent.bottom
+  }
+  height: __style.toolbarHeight
+  color: __style.forestColor
+
+  GridView {
+    id: buttonView
+
+    anchors.fill: parent
+    leftMargin: 20 * __dp
+    rightMargin: 20 * __dp
+    cellHeight: __style.toolbarHeight
+    cellWidth: Math.floor((control.width - leftMargin - rightMargin) / control.model.count)
```

Comment:
This does not seem to be correct. I can not see any left/right padding in the design :) It could have been there in some previous versions.

<img width="530" alt="image" src="https://github.com/MerginMaps/mobile/assets/22449698/c4d0ea5d-c964-499a-b9c7-920c162838da">